movies/11.sql

```diff
@@ -0,0 +1 @@
+SELECT movies.title FROM people JOIN stars ON people.id = stars.person_id JOIN movies ON stars.movie_id = movies.id JOIN ratings ON movies.id = ratings.movie_id WHERE people.name = "Chadwick Boseman" ORDER BY rating DESC LIMIT 5;
```

Comment:
```suggestion
SELECT movies.title
    FROM people 
    JOIN stars 
       ON people.id = stars.person_id 
    JOIN movies 
       ON stars.movie_id = movies.id 
    JOIN ratings
       ON movies.id = ratings.movie_id
       WHERE people.name = "Chadwick Boseman"
       ORDER BY rating DESC LIMIT 5;
    
```

stick to the convention you used on the `fiftyville/log.sql` file. this is also for the other `.sql` files.